docs/tools/terraform/v1v2.rst

```diff
@@ -0,0 +1,216 @@
+them in the Command Line prompt). For example from
+`here <https://github.com/aiven/aiven-examples/tree/aiven-terraform-v1.3.5>`__
+or use the following example from the blog post of the v2 release:
+
+::
+
+    terraform {
+      # go install github.com/aiven/terraform-provider-aiven@v1.3.5
+      # mkdir -p $PWD/terraform.d/plugins/linux_amd64/
+      # cp $GOPATH/bin/terraform-provider-aiven \
+      #   $PWD/terraform.d/plugins/linux_amd64/terraform-provider-aiven_v1.3.5
+      required_providers {
+        aiven = "1.3.5"
+      }
+    }
+
+    variable "aiven_api_token" {
+      type = string
+    }
+
+    provider "aiven" {
+      api_token = var.aiven_api_token
+    }
+
+    data "aiven_project" "tf" {
+      project = "terror"
+    }
+
+    resource "aiven_service" "kf" {
+      project                 = aiven_project.tf.project
+      cloud_name              = "google-europe-west1"
+      plan                    = "business-4"
+      service_name            = var.kafka_svc
+      service_type            = "kafka"
+      maintenance_window_dow  = "saturday"
+      maintenance_window_time = "10:00:00"
+
+      kafka_user_config {
+        kafka_connect   = true
+        kafka_rest      = true
+        kafka_version   = "2.8"
+        schema_registry = true
+
+        kafka {
+          group_max_session_timeout_ms = 70000
+          log_retention_bytes          = 1000000000
+          auto_create_topics_enable    = true
+        }
+        public_access {
+          kafka_rest    = true
+          kafka_connect = true
+          prometheus    = true
+        }
+      }
+    }
+
+``terraform init``
+
+``terraform plan``
+
+``terraform apply``
+
+Upgrade Terraform
+-----------------
+
+Kick things off with 0.13
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are using more providers than Aiven provider and/or your module
+syntax is 0.12, then you need to upgrade your modules and providers
+first by installing Terraform v0.13.x (i.e. 0.13.7):
+``tfenv install 0.13.7 && tfenv use 0.13.7``
+
+You will likely have an existing state file, by running:
+``terraform state replace-provider registry.terraform.io/-/aiven registry.terraform.io/aiven/aiven``
+you will replace old Aiven terraform provider references to the new
+format.
+
+Also update ``required_version`` from ``>= 0.12`` to ``>= 0.13`` in your
+``versions.tf`` file if you have any.
+
+We can remove the old Terraform folder ``rm -rf ~/.terraform.d``
+
+Edit the providers block of your script to include the latest version of
+the Aiven Terraform Provider (v2.3.1 as of 5th November 2021)
+
+::
+
+    terraform {
+      required_providers {
+        aiven = {
+          source = "aiven/aiven"
+          version = "2.3.1"
+        }
+      }
+    }
+
+After that you can run: ``terraform 0.13upgrade`` to see fixes
+recommended by Hashicorp. More information
+`here <https://www.terraform.io/upgrade-guides/0-13.html>`__.
+
+Bump it to the latest
```

Comment:
Here we have "Bump it" but does it mean "Upgrade Terraform"?

Reply:
Yep, it implies updating TF version to the latest available.

Reply:
Let's reword the title to make it clear for anyone skim reading the navigation headings.